netinstall: Add pipewire-jack

crian opened the pull request:
Adding pipewire-jack because jack is a dependency of some applications, otherwise jack audio server will be installed

adriaandegroot committed:
[finishedq] Point to the Calamares domain rather than GH

ArrayBolt3 committed:
[bootloader] Add hybrid GRUB installation support

Some users may want to install their system in a way that it can be
booted with both BIOS and UEFI firmware. This requires installing both
BIOS and UEFI bootloaders. This commit adds the ability to do this
with GRUB.

ArrayBolt3 committed:
[partition] Add hybrid bootloader compatible partitioning mode

Some users may want to install their system in a way that it can be
booted with both BIOS and UEFI firmware. This requires installing both
BIOS and UEFI bootloaders. This commit adds the ability to
automatically partition a disk in a way that is compatible with this
setup.

adriaandegroot committed:
PolicyKit: Replace GH by Calamares' own domain

adriaandegroot committed:
[calamares] Update documentation and donation URLs

The ones in branding are just an example.

adriaandegroot committed:
[man] Adjust URL and mention the session.log

adriaandegroot committed:
[license] Repair explanatory URLs

The screenshot is no longer available, adapt KaOS
repository URL.

1Naim commented:
> jack is a dependency of some applications

such as?

crian commented:
ffmpeg, ffmpeg4.4, portaudio,...

vnepogodin commented:
I don't think the conflict was fixed
https://github.com/CachyOS/cachyos-calamares/commit/e7c8879f23f30ba4a1c07b73ff2d86add986dd18

crian commented:
I think the conflict arises because jack audio server was already installed as a dependency earlier in the installation process

1Naim commented:
> I think the conflict arises because jack audio server was already installed as a dependency earlier in the installation process

Yeah, I guess so. Not really sure the best way to go about this really. I don't think adding the whole `pipewire` stack is ideal, and swapping between the two impl. isn't going to be clean either.

vnepogodin commented:
> I think the conflict arises because jack audio server was already installed as a dependency earlier in the installation process

yep, exactly that. I just meant that I don't think adding that to netinstall is possible.
a way to fix that was moving netinstall before chwd, and the other one is adding pipewire stack to the pacstrap meaning it cannot be customized by the user

crian commented:
So the pipewire stuff would have to be one of the first things installed.

ArrayBolt3 committed:
[grubcfg] Fix /etc/default/grub.d config file name

Under at least Debian, files under /etc/default/grub.d that don't carry
a file name extension of '.cfg' are ignored. Change the filename used
when grubcfg's prefer_grub_d option is set to 'true' to
00calamares.cfg, so that it is detected properly.

adriaandegroot committed:
Merge pull request #2422 from ArrayBolt3/arraybolt3/multi-firmware-boot

Hybrid bootloader (BIOS+UEFI) installation support

adriaandegroot committed:
Merge pull request #2445 from ArrayBolt3/arraybolt3/fix-grub-d

[grubcfg] Fix /etc/default/grub.d config file name

Neixen911 committed:
[users] Add a choice to display autologin checkbox

By default, the autologin checkbox is display to the user and he can
choose if he wants to enable it or not. By adding a variable to display
the checkbox or not, this checkbox can be not displaying to the user and
avoid him to change it.

ptr1337 committed:
bootloader: Drop bootloader.orig file

Signed-off-by: Peter Jung <admin@ptr1337.dev>

dalto8 committed:
Merge pull request #2448 from ptr1337/drop-orig

bootloader: Drop bootloader.orig file

Neixen911 committed:
[users] Add a setting to displayAutologin

To add this new setting optionnal, a variable has need to be created
in the file users.conf. This variable can be modified and it changes
directly if the autologin checkbox is show or not.

adriaandegroot committed:
fixup-policykit

adriaandegroot committed:
*: update documentation links from wiki to site

Neixen911 committed:
[users] Set displayAutologin to true by default

To match the existing behavior, the setting displayAutoLogin have to
be set to true bt default. This new setting is added to
users.schema.yaml too.

adriaandegroot committed:
CI: remove AppImage script

This was a possibly-interesting development 5 years ago,
but it isn't used at all.

adriaandegroot committed:
CI: Update release-announcement instructions

I don't see a releases page on Codeberg, leave the
instructions generic for now.

adriaandegroot committed:
Documentation: update URLs

adriaandegroot committed:
CMake: update URLs and domain information

adriaandegroot committed:
Merge pull request #2447 from Neixen911/calamares

[users] Add a choice to display autologin checkbox

adriaandegroot committed:
Documentation: update more references to the forge

adriaandegroot committed:
Merge branch 'codeberg-migration' into calamares

adriaandegroot committed:
CI: enable a simple `build.sh -` build-defaults-simply script

adriaandegroot committed:
[users] Add a configurable group for no-password login

crian commented:
@ptr1337 what is the status on this one?

dalto8 committed:
[initcpiocfg] Remove crc32c_intel which is no longer needed

Commit:
Merge pull request '[initcpiocfg] Remove crc32c_intel which is no longer needed' (#2436) from remove-crc32c into calamares

Reviewed-on: https://codeberg.org/Calamares/calamares/pulls/2436

kel-mo committed:
Add erofs support to unpackfsc module.

Signed-off-by: kelmo <kelvmod@gmail.com>

kel-mo committed:
Update copyright text of ErofsRunner.{h,cpp} to correctly reflect contributer.

Signed-off-by: kelmo <kelvmod@gmail.com>

kel-mo committed:
When inode stats cannot be obtained via dump.erofs -s, report the full command including source file.

Signed-off-by: kelmo <kelvmod@gmail.com>

kel-mo committed:
Co-locate and give the progress reporting variable names more descriptive names.

Signed-off-by: kelmo <kelvmod@gmail.com>

kel-mo committed:
Add verbose descriptions of EROFS and SquashFS to unpackfsc.conf.

Signed-off-by: kelmo <kelvmod@gmail.com>

kel-mo committed:
Update modules/unpackfsc/UnsquashRunner

Co-locate and give the progress reporting variable names more descriptive names.
When inode stats cannot be obtained via unsquashfs -s, report the full command
including source file.

Signed-off-by: Kel Modderman <kelvmod@gmail.com>

kel-mo committed:
Fix typo in variable name: s/updateUIEveryNLinex/updateUIEveryNLines/

Signed-off-by: Kel Modderman <kelvmod@gmail.com>

kel-mo committed:
Fix typo in variable name: s/updateUIEveryNLinex/updateUIEveryNLines/

Signed-off-by: Kel Modderman <kelvmod@gmail.com>

Commit:
Update Readme

fixes broken links caused by migration, and change matrix link to matrix.to

Commit:
Merge pull request 'Add erofs support to unpackfsc module.' (#2444) from kelmo/calamares:erofs-unpackfsc into calamares

Reviewed-on: https://codeberg.org/Calamares/calamares/pulls/2444
Reviewed-by: Adriaan de Groot <groot@kde.org>

Commit:
Merge pull request 'Update modules/unpackfsc/UnsquashRunner' (#2445) from kelmo/calamares:squashfs-unpackfsc into calamares

Reviewed-on: https://codeberg.org/Calamares/calamares/pulls/2445

Commit:
Merge pull request 'Update Readme' (#2446) from nebras/calamares:readme into calamares

Reviewed-on: https://codeberg.org/Calamares/calamares/pulls/2446

adriaandegroot committed:
AUTHORS: update authors list

adriaandegroot committed:
CMake: pre-release housekeeping

Switch the minor number to 4, just as an indicator "this is from
Codeberg".

adriaandegroot committed:
i18n: [calamares] Automatic merge of Transifex translations

adriaandegroot committed:
i18n: [python] Automatic merge of Transifex translations

adriaandegroot committed:
CMake: switch default build to Qt6

adriaandegroot committed:
CMake: stop treating CMakeLists.txt as a module subdirectory

Filter out some known non-a-directory files. This makes
CMake-time output more readable.

adriaandegroot committed:
users: repair test

In 6ff2ca450b4b8, a new group was added to the example
configuration, but the test was not updated.

adriaandegroot committed:
CI: check GPG key availability early

adriaandegroot committed:
Changes: introduce a new changelog file for 3.4

adriaandegroot committed:
CMake: post-release housekeeping

adriaandegroot committed:
CMake: pre-release housekeeping

adriaandegroot committed:
i18n: [calamares] Automatic merge of Transifex translations

adriaandegroot committed:
i18n: update translations

- Removed ro_RO (no translators, no strings, overlap with language ro)
- Removed bqi (no translators, no strings)

Commit:
CMake: make Boost::python components configurable

SixK committed:
Protect regexp with raw string r"" instead of "" to avoid synthax
warning : invalid escape sequence '\$'

adriaandegroot committed:
i18n: [python] Automatic merge of Transifex translations

adriaandegroot committed:
i18n: [calamares] Automatic merge of Transifex translations
